src/core/user/tld.ts

```diff
@@ -47,3 +47,11 @@ export function tld(url: URL): string | undefined {
     }
   }
 }
+
+export function tld(url: URL): string | undefined {
```

Comment:
Can we rewrite this to be more explicit about why and where we're silently swallowing errors? For example, just wrapping the specific part of tld that we expect to be unsafe in a try/catch, leaving a comment, etc.

Reply:
I added a comment on why we throw the error. Unfortunately `cookie.get()` itself will throw an error if the library is ran in a sandboxed environment. Unfortunately `Navigator.cookieEnabled` doesn't give the correct behavior in IE/Safari/old Edge inside iframes so we can't rely on that either.